core/invoker/src/main/scala/whisk/core/containerpool/ContainerPool.scala

```diff
@@ -132,14 +133,22 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
                   .headOption
                   .map(_ =>
                     takePrewarmContainer(r.action)
-                      .map(container => (container, "recreatedPrewarm"))
-                      .getOrElse(createContainer(r.action.limits.memory.megabytes.MB), "recreated")))
+                      .map(container => (container, "recreatedPrewarm", container._2.activeActivationCount))
+                      .getOrElse(createContainer(r.action.limits.memory.megabytes.MB), "recreated", 0)))
           } else None
 
         createdContainer match {
-          case Some(((actor, data), containerState)) =>
-            busyPool = busyPool + (actor -> data)
-            freePool = freePool - actor
+          case Some(((actor, data), containerState, activeActivations)) =>
```

Comment:
I think `activeActivationCount` in `data` which is `ContainerData` == `activeActivations`. In that case we can avoid extracting `activeActivationCount` in steps before

Reply:
will try to remove that 👍